### PR TITLE
Bug 1851675: Include graphql schema in downstream docker build

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: Dockerfile.product [hunk 1]
```diff
@@ -39,6 +39,7 @@ FROM openshift3/origin-base
 RUN mkdir -p /opt/bridge/bin
 COPY --from=gobuilder /go/src/github.com/openshift/console/bin/bridge /opt/bridge/bin
 COPY --from=nodebuilder /opt/app-root/src/frontend/public/dist /opt/bridge/static
+COPY --from=build /go/src/github.com/openshift/console/pkg/graphql/schema.graphql /pkg/graphql/schema.graphql
```
Comment (jhadvig) [inline Dockerfile.product:42]:
```suggestion
COPY --from=gobuilder /go/src/github.com/openshift/console/pkg/graphql/schema.graphql /pkg/graphql/schema.graphql
```

Comment (rawagner) [reply Dockerfile.product:42]: github is having some issues. I've updated the code a while ago https://github.com/rawagner/console/commit/f91a7abf57a6ec28479cfd820ab94e29b09abf7f but its not reflected here yet. I think we can just omit `--from`

Comment (jhadvig) [reply Dockerfile.product:42]: yeah I was having issues with commenting on the PR.

> I think we can just omit --from

I would rather follow the instructions that we have in the `Dockerfile` from your previous PR so they dont differ

Comment (rawagner) [reply Dockerfile.product]: ok, added `--from=gobuilder`

**Pull request merged**